### PR TITLE
Introduce Fixed elements

### PR DESCRIPTION
**Pull request by mtomov**
* add deprecation note on Cells

image

image

@tvdeyen Thanks for tips on Slack!

### CODE REVIEW

File: guides/cells.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ next: /elements.html
 
 # Cells
 
+::: tip Deprecation note
+Cells have now been superseded by [fixed elements](/elements.html#render-a-group-of-elements-on-a-fixed-place-on-the-page).
```
Comment (tvdeyen) [inline guides/cells.md]: Maybe add a note about the Version of Alchemy where this has been introduced
```suggestion
With Alchemy 4.2 cells have now been superseded by [fixed elements](/elements.html#render-a-group-of-elements-on-a-fixed-place-on-the-page).
```

File: guides/elements.md [hunk 2]
```diff
@@ -327,6 +327,40 @@ A common use case is to have global pages for header and footer parts.
 
 Given global pages with `page_layout` "header" and "footer".
 
+### Render a group of elements on a fixed place on the page
+
+Often you have a separate section on a page like a sidebar where you'd like to place a few different elements together. If you tag those elements as `fixed: true` in `elements.yaml`, then they'll be easy to grab from the frontend, and the same will be cleanly separated in a new tab in the admin elements section.
```
Comment (tvdeyen) [inline guides/elements.md]: The file extension needs to be `.yml`

```suggestion
Often you have a separate section on a page like a sidebar where you'd like to place a few different elements together. If you tag those elements as `fixed: true` in `elements.yml`, then they'll be easy to grab from the frontend, and the same will be cleanly separated in a new tab in the admin elements section.
```

File: guides/elements.md [hunk 3]
```diff
@@ -327,6 +327,40 @@ A common use case is to have global pages for header and footer parts.
...
+~~~
+
+~~~ erb
+<% @page.fixed_elements.each do |element| %>
+  <%= render_element(element) %>
+<% end %>
+~~~
+
+::: tip INFO
+Fixed elements replace the concept on [Cells](/cells.html) in previous versions of Alchemy.
```
Comment (tvdeyen) [inline guides/elements.md]: Here as well a note on the version would be great

```suggestion
Fixed elements replace the concept on [Cells](/cells.html) in Alchemy versions prior to 4.2.
```

**Comment by mtomov** [comment]
Many thanks for the suggestions as well!

**Comment by tvdeyen** [comment]
Thank you! Maybe we can add the `fixed` setting to the element settings as well?

**Comment by mtomov** [comment]
Makes sense, yeah, thanks, just added and squashed all!

**Comment by tvdeyen** [comment]
Thank you so much

**Pull request merged**